sormas-rest/src/main/java/de/symeda/sormas/rest/CaseResource.java

```diff
@@ -77,7 +77,7 @@
 	@POST
 	@Path("/push")
 	public List<PushResult> postCases(@Valid List<CaseDataDto> dtos) {
-		return savePushedDto(dtos, FacadeProvider.getCaseFacade()::saveCase);
+		return savePushedDto(dtos, dto -> FacadeProvider.getCaseFacade().saveCase(dto));
```

Comment:
Strange refactoring - the `::` notation should actually be the preferred one by the IDE